releases/index.rst

```diff
@@ -0,0 +1,33 @@
+Borg release series
+===================
+
+`« back </>`_
+
+Borg 1.1
+--------
+
+Borg 1.1 is the current **development** branch of Borg and is in the **beta** phase.
+It is not recommended to use beta software in a production environment except
+for evaluation and testing.
+
+- `Documentation <https://borgbackup.readthedocs.io/en/latest/>`__
+- `Installation <https://borgbackup.readthedocs.io/en/latest/installation.html>`__
+- `Downloads <https://github.com/borgbackup/borg/releases/>`__
+
+Borg 1.0
+--------
+
+The current **stable** series is Borg 1.0, with the initial 1.0.0 release on 5th March 2016.
+The latest release in this series is Borg 1.0.10, released on 13th February 2017.
+This is the series shipped by most Linux distributions.
+
+- `Documentation <https://borgbackup.readthedocs.io/en/stable/>`__
+- `Installation <https://borgbackup.readthedocs.io/en/stable/installation.html>`__
+- `Downloads <https://github.com/borgbackup/borg/releases/latest>`__
```

Comment:
Yes, these will need to be shuffled around when the stable series changes. 1.1 latest->stable, 1.0 stable->1.0-stable (? 1.0-maint is IIRC the unreleased branch).